tests/test_disk_io.py

```diff
@@ -127,3 +129,33 @@ def test_get_loader_training_fail():
         torch_loader_train, torch_loader_val = test_loader.get_loader(
             split_size=1, batch_size=64
         )
+
```

Comment:
Also, it will be good to test the data loader returning what we want, I think here we have only tested that the class variables are correct (which is great!) but we won't catch things like the data loader not returning the right size of data, or selecting correctly the labels.

Reply:
that's a solid point! I checked it but realised it was only on a local function, I'll make that into assertions here